openvpn-install.sh

```diff
@@ -242,8 +261,8 @@ else
 	echo "   7) Google (Anycast: worldwide)"
 	echo "   8) Yandex Basic (Russia)"
 	echo "   9) AdGuard DNS (Russia)"
-	while [[ $DNS != "1" && $DNS != "2" && $DNS != "3" && $DNS != "4" && $DNS != "5" && $DNS != "6" && $DNS != "7" && $DNS != "8" && $DNS != "9" ]]; do
-		read -p "DNS [1-9]: " -e -i 1 DNS
+	until [[ "$DNS" =~ ^[0-9]+$ ]] && [ "$DNS" -ge 1 -a "$DNS" -le 9 ]; do
```

Comment:
Why the `-a` here? 🤔

Reply:
if [ $DNS -gt 1 ] && [ $DNS -lt 9 ]
    #  Returns true if both condition1 and condition2 hold true...

It’s shorter version of “&&” used inside.

Reply:
if [ $DNS -gt 1 ] && [ $DNS -lt 9 ]
    #  Returns true if both condition1 and condition2 hold true...

It’s shorter version of “&&” used inside.